projects/canopy/src/lib/shared/breakpoints.interface.ts

```diff
@@ -0,0 +1 @@
+export type Breakpoints = 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
```

Comment:
@elenagarrone @andy-polhill - I've removed the responsive module which combined `showAt` and `hideAt`, but wanted to put this interface somewhere - it's used by both directives. First thought was this new `src/lib/shared` directory. What do you think?

Reply:
makes sense 👍  works for me.